index.html

```diff
@@ -1,29 +1,183 @@
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--
-
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
-
-    -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Benjamin Scott Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="avatar.png">
+  </head>
+  <body>
+    <!--The top part of the header-->
+    <!--I used hero class names but its really part of the header-->
+    <!--This section of the header could also count as a hero section-->
+    <section class="hero">
+      <div class="herocontainer">
+        <div class="heroimage">
+          <img src="avatar.png" alt="Logo">
+        </div>
+        <div class="herotext">
+          <h1>Benjamin Scott</h1>
+          <p>Concept Artist</p>
+          <p>Contact, Canada</p>
+          <p>Contact me:</p>
+          <a href="/page"><p>bscott1@outlook.com</a> | 613-444-8910</p>
+          <a href="/page"><p>linkedin.com/thebenjaminscott</a></p>
+        </div>
+      </div>
+    </section>
+    
+
+    <!--Header, there is a rightline class on the first 3 div boxes, adds a gray divider line-->
+    <section class="header" id="header">
+      <a href="#Portfolio"><div class="rightline"><p>Portfolio</p></div></a>
+      <a href="#About"><div class="rightline"><p>About</p></div></a>
+      <a href="#Contact"><div class="rightline"><p>Contact</p></div></a>
+      <a href="/page"><div><p>Resume</p></div></a>
+    </section>
+
+
+
+    <script>
+      // Reference code taken from:
+      // https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_sticky_header
+
+      window.onscroll = function() {scroll()};
+      //Everytime the user scrolls, the function called scroll will be called
+
+      var header = document.getElementById("header");
+      var headerY = header.offsetTop;
+      //The header is declared so it can be modified
+      //The Y coordinate of the header is put into a variable
+
+
+      function scroll(){
+        if (window.pageYOffset > headerY){
+          header.classList.add("headersticky");
+        } else {
+          header.classList.remove("headersticky");
+        }
+        // If the scrolling reaches the top of the header, the header gets a class of headersticky,
+        // which makes it stay at the top
+      }
+    </script>
+
+    
+    <!--Images in a grid-->
+    <!--I tried using a php for loop for this but I didnt know how to import php into this file/visual studio code-->

```

Comment:
PHP is a server side language and so won't run in GitHub Pages with just serves static content. 

You could do this in JavaScript pretty easily! 

Have an array of images and then run a for loop and create / add elements to the list.